packages/main/src/Link.js

```diff
@@ -127,10 +127,46 @@ const metadata = {
 			defaultValue: "",
 		},
 
+		/**
+		 * Defines the aria-haspopup value of the component.
+		 *
+		 * @type String
+		 * @defaultvalue undefined
+		 * @since 1.0.0-rc.15
+		 * @private
+		 */
+		 ariaHaspopup: {
+			type: String,
+			defaultValue: undefined,
+		},
+
+		/**
+		 * Defines the accessibility role of the component.
+		 * @defaultvalue ""
+		 * @private
+		 * @since 1.0.0-rc.15
+		 */
+		role: {
```

Comment:
The text "More button menu" is read out twice upon focusing the `ui5-link` with overflowed items.
This is because of this attribute being named `role`. The screen reader reads the role and the label twice. Once from the custom element and once from the internal shadow dom element. **If we rename this property to `accRole` the announcment is correct.**

Reply:
Maybe **accessibleRole**, to be consistent with the List and Panel components, where this is how it is named.

Reply:
Renamed to "accessibleRole"